tnz/zti.py

```diff
@@ -182,7 +182,7 @@ def cmdloop(self, intro=None):
                 self.__shell_mode()
                 print("\nKeyboardInterrupt")
 
-            except BaseException:
+            except Exception:
```

Comment:
This particular use of `BaseException` wants to remain `BaseException`. The intent here is to ensure that `postloop()` is called once `cmdloop()` may process something. For instance, if `SystemExit` is raised by `cmdloop()`, `postloop()` should still be called.